### ISSUE TITLE
Set up Labs directors check-ins before/after club meetings

### ISSUE THREAD

**Issue by MaxWofford** [hackedu/meta#157]
This can be closed when all check-ins with Labs directors (before and after meetings) for this week are scheduled on the Labs calendar(#156).

**Comment by jonleung** [comment]
@MaxWofford, I've decided to not add this to the labs calendar because I may change when I checkin with people and because calendars are hard to update, the information most likely won't be accurate in the calendar.

Instead, I've made [this spreadsheet](https://docs.google.com/spreadsheets/d/1_K9uZg0A7ZoMRWqSxDoVgKSe64IJ7DZJKpVOZapdVKc/edit?usp=sharing) that tentatively outlines when I'm going to interact with people.

**Club & Recap**: "Club" means they have a club meeting. "Recap" means that I'm going to recap the club meeting with them for a bit and poke them for their recap.

**Ensure Plan**: means checking in with club leaders to see what they're planning to do for their club meeting.

**Curriculum Checking**: checking in to make sure to let Labs leaders know what the most up to date curriculum is.
